### PR TITLE
Download automatically altcoin datas

### PR DESCRIPTION
**Pull request by dertione**

### CODE REVIEW

File: freqtrade/tests/testdata/download_backtest_data.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/python3.6
```
Comment (gcarq) [inline freqtrade/tests/testdata/download_backtest_data.py]: Use `#!/usr/bin/env python3` as shebang

File: freqtrade/tests/testdata/download_backtest_data.py [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+from urllib.request import urlopen
+
+currencies = ["ok","neo","dash","etc","eth","snt"]
+
+for cur in currencies:
+	url = 'https://bittrex.com/Api/v2.0/pub/market/GetTicks?marketName=BTC-'+cur+'&tickInterval=fiveMin'
+	x = urlopen(url)
+	data = x.read()
+	str2 = str(data,'utf-8')
+	with open("btc-"+cur+".json", "w") as fichier:
```
Comment (gcarq) [inline freqtrade/tests/testdata/download_backtest_data.py]: Use english variable names

File: freqtrade/tests/testdata/download_backtest_data.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+"""This module download automatically json data of crypto currencies in bittrex"""
+#!/usr/bin/env python3
```
Comment (gcarq) [inline freqtrade/tests/testdata/download_backtest_data.py]: shebang has to be in the first line

File: freqtrade/tests/testdata/download_backtest_data.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+from urllib.request import urlopen
+
+CURRENCIES = ["ok", "neo", "dash", "etc", "eth", "snt"]
+
+for cur in CURRENCIES:
+	url1 = 'https://bittrex.com/Api/v2.0/pub/market/GetTicks?marketName=BTC-'
+	url2 = '&tickInterval=fiveMin'
+	url = url1+cur+url2
+	x = urlopen(url)
```
Comment (gcarq) [inline freqtrade/tests/testdata/download_backtest_data.py]: Please use [requests](http://docs.python-requests.org/en/master/) it is more human readable, e.g.:
```
params = {
    'marketName': 'BTC-',
    'tickInterval': 'fiveMin',
}
res = requests.get('https://...', params=params)
data = res.json()
...
```

**Comment by dertione** [comment]
the file has been updated from branch Patch-2

**Pull request merged**